src/main/java/fr/foodlimit/api/scheduled/ScheduledTasks.java

```diff
@@ -0,0 +1,84 @@
+package fr.foodlimit.api.scheduled;
+
+import fr.foodlimit.api.food.FoodService;
+import fr.foodlimit.api.shared.models.Food;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Iterator;
+
+@Component
+public class ScheduledTasks {
+  @Autowired
+  FoodService foodService;
+
+  @Value("${onesignal.url}")
+  private String url;
+  @Value("${onesignal.api-key}")
+  private String apiKey;
+  @Value("${onesignal.app-id}")
+  private String appId;
+
+  private static final Logger log = LoggerFactory.getLogger(ScheduledTasks.class);
+
+  @Scheduled(cron="0 0 19 * * *")
+  public void notifyAllUsersWithExpiredFoodsIn3Days() throws IOException {
+    System.out.println(url);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)